src/Test/QuickCheck/Gen.purs

```diff
@@ -229,6 +229,12 @@ evalGen = evalState <<< unGen
 sample :: forall a. Seed -> Size -> Gen a -> Array a
 sample seed sz g = evalGen (vectorOf sz g) { newSeed: seed, size: sz }
 
+-- | Generate a single value using a randomly generated seed.
+generate :: forall a. Gen a -> Effect a
+generate gen = do
+  seed <- randomSeed
+  pure $ evalGen gen { newSeed: seed, size: 1 }
```

Comment:
I think 1 is probably too small for the `size` parameter, as that will probably restrict the values which can come out; we should pick a `size` large enough to ensure that most/all of the values this generator can produce can be returned here. The existing `randomSample` uses a size of 10, for instance. I'm not sure whether 10 is a good default here though. Can you give this a try with some common generators and see what you get?

I also think it's not ideal that the name `generate` doesn't indicate that this function has anything to do with the existing `randomSample` function, even though they're very similar. I think I'd prefer `singleSample` or `sampleOne` or something. Does anyone else have ideas?

Reply:
A size of 1 is definitely too small, that'd result in a 1-element array, or one level of structure in a sized recursive generator.

I think 10 seems reasonable, and it won't override a `Gen` that is `resized` to more or less than that too. A quick check of some common generators with that size would be good just to double check, but I'd probably have chosen 10 if I were writing it too.

Reply:
> I also think it's not ideal that the name generate doesn't indicate that this function has anything to do with the existing randomSample function, even though they're very similar. I think I'd prefer singleSample or sampleOne or something. Does anyone else have ideas?

What about `randomOne`? If one starts typing `ran` for either one of these functions, it'll show all three of them in the dropdown.

Reply:
I don't really like `randomOne` because I think the "sample" is more important than the "random" in `randomSample`; what you're doing is sampling from the generator. Perhaps `randomSampleOne`? That way it is clearly a variant of `randomSample`, and they'll still all turn up in the completion dropdowns.

Reply:
Done.